ZipBuilder/Sources/ZipBuilder/FrameworkBuilder.swift

```diff
@@ -444,7 +492,9 @@ struct FrameworkBuilder {
     let xcframework = packageXCFramework(withName: framework,
                                          fromFolder: frameworkDir,
                                          thinArchives: thinArchives,
-                                         moduleMapContents: moduleMapContents)
+                                         moduleMapContents: moduleMapContents.replacingOccurrences(
```

Comment:
Nit: Will moving `moduleMapContents` calculation to a separate method improve readability?